{
  "id": "MAL-2024-5787",
  "modified": "2024-10-24T01:01:59Z",
  "published": "2024-06-25T13:40:47Z",
  "summary": "Malicious code in python-sdkcore (PyPI)",
  "details": "\n---\n_-= Per source details. Do not edit below this line.=-_\n",
  "affected": [
    {
      "package": {
        "ecosystem": "PyPI",
        "name": "python-sdkcore",
        "purl": "pkg:pypi/python-sdkcore"
      },
      "versions": [
        "1.0.1"
      ],
      "database_specific": {
        "source": "https://github.com/ossf/malicious-packages/blob/main/osv/malicious/pypi/python-sdkcore/MAL-2024-5787.json"
      }
    }
  ]
}